sdk/managednetworkfabric/Azure.ResourceManager.ManagedNetworkFabric/src/Customize/NetworkFabricInternetGatewayData.cs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System.ComponentModel;
+using System.Net;
+using Azure.ResourceManager.Models;
+
+namespace Azure.ResourceManager.ManagedNetworkFabric
+{
+    /// <summary>
+    /// A class representing the NetworkFabricInternetGateway data model.
+    /// The Internet Gateway resource definition.
+    /// </summary>
+    public partial class NetworkFabricInternetGatewayData : TrackedResourceData
+    {
+
+        /// <summary> IPv4 Address of Internet Gateway. </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
Add `Obsolete` attribute as well since the previous type will not work anymore, it should be string.